core/src/main/java/org/infinispan/commands/AbstractLocalFlagAffectedCommand.java

```diff
@@ -12,15 +11,23 @@
  */
 public abstract class AbstractLocalFlagAffectedCommand implements LocalFlagAffectedCommand {
 
-   protected Set<Flag> flags;
+   private long flags = 0;
 
    @Override
-   public Set<Flag> getFlags() {
+   public long getFlagsBitSet() {
```

Comment:
the getFlags is not removed. It is implemented in the interface.
Yes, it was the only way I had to copy the flags to another commands (e.g. cluster get to lock control)